intro/notebooks-intro/notebooks-intro.ipynb

```diff
@@ -0,0 +1,311 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# Generate data\n",
+    "x = np.linspace(0, 10, 100)\n",
+    "y = np.sin(x)\n",
+    "\n",
+    "# plot\n",
+    "#plt.plot(x, y, linewidth=2.0)\n",
+    "#plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "8. Enable the debug mode by clicking the `Enable debugger` _bug_ icon at the top right of this notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "9. Select the following cell, and then set a breakpoint on line 3.\n",
+    "Then press `Ctrl+Enter` to verify that the debugger stops at line 3."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = 4\n",
+    "b = 5\n",
+    "y = 2 * x + b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "10. Stop the debugger by clicking the _bug_ icon again."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "11. Set the following cell to markdown type.\n",
+    "To do this, make sure that the cell is not in edit mode, then press `m`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "This is intended to be markdown text and should not print a python error."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "12.Set the following cell to code type.\n",
+    "To do this, make sure that the cell is not in edit mode, then press `y`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "result = 2 + 2\n",
+    "print(f'2 + 2 is {result}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "13. Create an ordered sequence from 1 to 5 by moving the following cells.\n",
+    "\n",
+    "    Click the `+` icon in the toolbar to add the missing element.\n",
+    "\n",
+    "    To move a cell, hover over the left margin of a cell until the pointer cursor changes to a move cursor.\n",
+    "    Then, drag the cell to the desired position.\n",
+    "\n",
+    "    To delete a cell, click the bin icon at the top-right corner of the cell."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "  1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "  3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "  5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "  4"
+   ]
```

Comment:
Again excuse my ignorance, this all looks a little different and I don't understand the technology. Is this supposed to appear in the preview as: 
1
3
5
4

Reply:
Yeap, that is part of the exercise. We present an unordered sequence and the student has to drag the numbers to the correct place.